xml/System.Net.Security/SslCertificateTrust.xml

```diff
@@ -43,10 +43,10 @@
         <Parameter Name="sendTrustInHandshake" Type="System.Boolean" />
       </Parameters>
       <Docs>
-        <param name="trustList">To be added.</param>
+        <param name="trustList">The store containig the trusted certificates.</param>
         <param name="sendTrustInHandshake">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <summary>Creates new SslCertificateTrust.</summary>
+        <returns>A collection of certificates replacing the default certificate trust.</returns>
```

Comment:
It doesn't return a collection of certificates, it returns the trust policy thing.

Reply:
Returns an object of this file's Type. So in that case, the `<returns>` value should look like the suggestion you provided here, right?:
https://github.com/dotnet/dotnet-api-docs/pull/7136/files#r706271785